core/src/apps/bitcoin/sign_tx/progress.py

```diff
@@ -1,31 +1,129 @@
-from trezor import ui, utils, workflow
+from typing import TYPE_CHECKING
+
+from trezor import ui, utils, wire, workflow
```

Comment:
Theoretically `wire` could be imported in `if __debug__` at the very bottom, it is not used in "non-debug" mode. (The reason is it occupies flash space even if it is unused).

Also `utils` and `workflow` are used only in a single function, so it can be imported there instead of top-level.

It is not so critical, each local import here would save around 5 bytes of flash space, but we might do these things consistently over the whole codebase - based on https://github.com/trezor/trezor-firmware/pull/2525

Reply:
https://github.com/trezor/trezor-firmware/pull/2507/commits/1d047e0b835765e76da42604237cea778cbcb131